django-stubs/apps/config.pyi

```diff
@@ -17,7 +18,8 @@ class AppConfig:
     models_module: str | None
     models: dict[str, type[Model]]
     def __init__(self, app_name: str, app_module: types.ModuleType | None) -> None: ...
-    default_auto_field: str
+    @cached_property
+    def default_auto_field(self) -> str: ...
```

Comment:
Heh this was just changed recently: https://github.com/typeddjango/django-stubs/pull/1541/files

Reply:
This looks like the cause of issue #1593

Reply:
Should we ignore stubtest and set s `str`, or try to cover both with `default_auto_field: cached_property[str] | str `?

Reply:
Just `str` seems fine, overengineering seems more likely to cause issues.